app/src/main/java/com/hover/stax/utils/Utils.kt

```diff
@@ -45,6 +45,11 @@ object Utils {
         return getSharedPrefs(c).getBoolean(key, false)
     }
 
+    fun getBooleanDefaultTrue(key: String?, c: Context): Boolean {
```

Comment:
Since this method is only used once, you can add a default argument to the `getBoolean()` method to reduce duplication.

Reply:
Done, thanks.